applicationconfig/client/src/main/java/com/azure/applicationconfig/ApplicationConfigCredentials.java

```diff
@@ -0,0 +1,58 @@
+package com.azure.applicationconfig;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Base64;
+
+class ApplicationConfigCredentials {
```

Comment:
One thing I often do when we make a class package-private (or non-public) is to just leave a // comment to make it clear that 1) this is intentional and 2) to help people realise that this is the case. It's all too easy to assume that all classes we look at are public, so a comment like 'package-private implementation class' might help clarify things to future readers of this code.